This patch causes the designer plugin to be built against the just-built
qscintilla, rather than trying to build against the system qscintilla
which may not be installed yet.

--- QScintilla-gpl-2.8.3.orig/designer-Qt4Qt5/designer.pro	2014-07-18 21:42:30.822408448 +0200
+++ QScintilla-gpl-2.8.3/designer-Qt4Qt5/designer.pro	2014-07-18 21:42:56.153048772 +0200
@@ -4,7 +4,10 @@
 TEMPLATE = lib
 TARGET = qscintillaplugin
 
-CONFIG += release plugin qscintilla2
+CONFIG += release plugin
+
+INCLUDEPATH += ../Qt4Qt5
+LIBS += -L../Qt4Qt5 -lqscintilla2
 
 greaterThan(QT_MAJOR_VERSION, 4) {
     QT += designer
